.gitattributes

```diff
@@ -0,0 +1,3 @@
+/doc export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
```

Comment:
Seems like the list of files to ignore is probably longer, including the build.sh in this PR, the typeMap files, and other files that may be added. I think it would be easier (more future-proof) to just specify the paths to include in the `git archive` command.